packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -24,13 +25,13 @@ import { contextConnect } from '../../ui/context';
 import { useBorderControlDropdown } from './hook';
 import { StyledLabel } from '../../base-control/styles/base-control-styles';
 import DropdownContentWrapper from '../../dropdown/dropdown-content-wrapper';
-
+import { VisuallyHidden } from '../../visually-hidden';
 import type { ColorObject } from '../../color-palette/types';
 import { isMultiplePaletteArray } from '../../color-palette/utils';
 import type { DropdownProps as DropdownComponentProps } from '../../dropdown/types';
 import type { ColorProps, DropdownProps } from '../types';
 
-const getAriaLabelColorValue = ( colorValue: string ) => {
+const getAriaDescriptionColorValue = ( colorValue: string ) => {
```

Comment:
I think this function should be renamed to remove anything to do with `aria`. Small point but this seems to just be stripped `var()` from any CSS vars am I right? In which case let's

1. Rename to better advertise it's purpose
2. Consider extracting to a file and adding some quick unit tests.

Reply:
I agree this could be made into a more generic utility function. Looks like it would [be useful elsewhere too](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/color-palette/index.tsx#L233). Would you be OK with this also being a follow-up? It's starting to expand the purpose of this PR a bit.

Reply:
I think we have a number of tasks that should happen as follow ups. Whilst I'm fine with them being follow ups I really think that whilst we're in the guts of this it would be good to leave the code in a better place than we found it if possible 🙏